web/src/app/administration/workbasket/details/distribution-targets/distribution-targets.component.ts

```diff
@@ -100,19 +107,88 @@ export class DistributionTargetsComponent implements OnChanges, OnDestroy {
 				}
 			});
 
-	}
+    this.orientationSubscription = this.orientationService.getOrientation().subscribe((orientation: Orientation) => {
+      this.calculateNumberItemsList();
+      this.getWorkbaskets();
+    });
+  }
+
+  calculateNumberItemsList() {
```

Comment:
What do you think about moving this metod into orientationService, using parameters and be as most minimal as you can.